src/components/Sum/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    <>
```

Comment:
You don't need to use Fragment here. 
The component returns everything wrapped inside the 'p' tag. 
It doesn't return multiple elements.